### PR TITLE
CQ-4353443: Add cypress ui tests

### PR DESCRIPTION
**Pull request by kronnox**
The goal of this PR is to have sample Cypress Tests published as part of the Archetype, so that customers which create new projects have them by default.

## Related Issue

- CQ-4353443

## How Has This Been Tested?

- Ran locally.
- Copied over generated files to CM env and ran pipeline.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: src/main/archetype/ui.tests.cypress/assembly-ui-test-docker-context.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    <includeBaseDirectory>false</includeBaseDirectory>
+    <formats>
+        <format>tar.gz</format>
+    </formats>
+    <fileSets>
+        <fileSet>
+            <directory>${basedir}</directory>
+            <includes>
+                <include>Dockerfile</include>
+                <include>testing.properties</include> <!-- opt-in test module in Cloud Manager -->
```
Comment (bisswanger) [inline src/main/archetype/ui.tests.cypress/assembly-ui-test-docker-context.xml]: Can you check on this line + testing.properties?
I think in the story we defined that by default there is no test opt-in yet after generation of the tests

Comment (bisswanger) [reply src/main/archetype/ui.tests.cypress/assembly-ui-test-docker-context.xml]: From JIRA:

> Cypress Tests are still opt-in, not enabled by default in the CM pipeline.

**Review by AlexRRR — APPROVED** [review]
Tested generation + test execution 👍  LGTM

**Review by dulvac — APPROVED** [review]
lgtm

**Pull request merged**